### PR TITLE
Update tag command to detect existing tags

### PR DESCRIPTION
**Pull request by hpkoh**

### CODE REVIEW

File: src/test/java/seedu/plannermd/logic/commands/tagcommand/AddDoctorTagCommandTest.java [hunk 1]
```diff
@@ -36,13 +38,15 @@
...
-    private final Tag tag = new Tag(VALID_TAG_FRIEND);
+    private final Tag tag = new Tag(VALID_TAG_DOCTOR);
 
     @Test
     void execute_validTagUnfilteredList_success() {
         Doctor firstDoctor = model.getFilteredDoctorList().get(INDEX_FIRST_PERSON.getZeroBased());
         Set<Tag> newTags = new HashSet<>(firstDoctor.getTags());
+        System.out.printf(String.valueOf(newTags));
         newTags.add(tag);
+        System.out.printf(String.valueOf(newTags));
```
Comment (pualixue) [inline src/test/java/seedu/plannermd/logic/commands/tagcommand/AddDoctorTagCommandTest.java:49]: Can remove the printf lines.

**Review by pualixue — COMMENTED** [review]
Looks good

File: src/test/java/seedu/plannermd/logic/commands/tagcommand/AddDoctorTagCommandTest.java (hunk 1 above)
Comment (hpkoh) [reply src/test/java/seedu/plannermd/logic/commands/tagcommand/AddDoctorTagCommandTest.java:49]: oops good catch

**Review by pualixue — APPROVED** [review]
LGTM

**Pull request merged**